apps/docs-snippets2/scripts/wrapper-fn.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+export const main = async (localNetworkUrl?: string): Promise<any | any[]> => {
```

Comment:
The wrapper method can optionally receive `localNetworkUrl` from outside.

If informed, that overrides `LOCAL_NETWORK_URL`.